.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,52 @@
+name: E2E over minikube
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+
+jobs:
+  e2eTests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ## we could run priviledgedMode this in serie instead of in //
+        ## PRO less total time, less expensive CONTRA slower to get results
+        privilegedMode: [ "normal", "unprivileged" ]
+        k8sVersion: [ "v1.15.12", "v1.16.15", "v1.17.14", "v1.18.12", "v1.19.4" ]
```

Comment:
wasn't tests 1.19 failing ?

Reply:
Seems like there is still some issue, but I think it was not related to the version